packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -458,6 +458,7 @@ const TRANSFORMS = {
       secureBoot: obj.platform.secureboot === 'true',
       suspendSr: link(obj, 'suspend_SR'),
       tags: obj.tags,
+      uefiMode: link(obj, 'uefi_mode'),
```

Comment:
Appears to be unused and also appears not to exist in the `VM` object. [Based on the XAPI doc](https://xapi-project.github.io/xen-api/classes/vm.html)
```suggestion
```